synapse/storage/events.py

```diff
@@ -591,10 +557,28 @@ def event_dict(event):
             ],
         )
 
-        if context.rejected:
-            self._store_rejections_txn(
-                txn, event.event_id, context.rejected
-            )
+        to_remove = set()
```

Comment:
`to_remove` is already declared at line 440. Is reinitialising it deliberate? needs explanation if so